scripts/datacube.py

```diff
@@ -423,28 +422,54 @@ def convert_attrs_and_coords_objects_to_str(data):
 @click.option(
     "--subset",
     required=False,
+    default="",
+    help="For debugging, subset x and y to this pixel window as a comma"
+    "separated string of 4 integers.",
+    type=str,
+)
+@click.option(
+    "--localpath",
+    required=False,
     default=None,
-    help="For debugging, subset x and y to this pixel window.",
+    help="If specified, this path will be used to write the tiles locally"
+    "Otherwise a temp dir will be used.",
+)
+@click.option(
+    "--dateranges",
+    required=False,
+    default="",
+    type=str,
+    help="Comma separated list of date ranges, each provided as yy-mm-dd/yy-mm-dd.",
```

Comment:
Is it year as 4 numbers (e.g. `2024`) or as 2 numbers (e.g. `24`)?

```suggestion
    help="Comma separated list of date ranges, each provided as yyyy-mm-dd/yyyy-mm-dd.",
```

Reply:
Good catch, I will update accordingly.